### PR TITLE
1st update of Kubeflow Roadmap for Kubeflow 1.6

### PR DESCRIPTION
**Pull request by jbottum**
@annajung @kimwnasptd @DomFleischmann I have created this PR as a quick way to update the Kubeflow Roadmap to include KF 1.6.   My view is that we can update this in a simple way and then add more details if needed.     Please review, improve as you see needed.   Thanks.   Josh

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.5 Release, Due: March 2022
+## Kubeflow 1.6 Release, Due: July 2022
+* Kubeflow 1.6 [milestones and timelines](https://github.com/kubeflow/community/pull/558)
+
```
Comment (annajung) [inline ROADMAP.md:5]: Adding a list of a few features shared by the WG during the April 26th Community meeting for WG pipelines, katib, training operators, and kserve.

```suggestion
### Major Features from each Working Group (note: Individual WG versions are independent of Kubeflow's)

#### Kubeflow Pipelines, v x.y.z
* V2 Preview: Support running pipeline in Argo-agnostic approach
* New DAG visualization based on Pipeline Template and MLMD
* SDK to change from `kfp.v2` to `kfp`
* KFP SDK CLI design

#### Katib, v x.y.z
* Support for Population based training
* Support JSON format for Metrics Collector
* Support for Kubernetes 1.22
* UI Enhancement to add a separate page for Trials and show Trial logs

#### Training Operator, v x.y.z
* PyTorch elastic training enhancements
* Elastic scheduling with Volano support
* Support for Kubernetes 1.22
* MPI Operator v2

#### Notebooks, Central Dashboard, Web Apps, Manifest, v x.y.z
* TBA

#### KServe, v x.y.z
* Knative 1.0 support and certified for KServe Serverless installation
* KServe 0.9 SDK with ModelServer API name changes and improvement for CloudEvent support
* New ServingRuntime and ClusterServingRuntime CRDs
* New ModelSpec in InferenceServices
* ModelMesh updates include support for multi-namespace reconciliation

```

Comment (jbottum) [reply ROADMAP.md:5]: @annajung line 21 - Volcano is missing a "c"

Comment (annajung) [reply ROADMAP.md:5]: fixed! thanks

Comment (annajung) [reply ROADMAP.md:5]: the notebook WG features have been added to the ROADMAP. Feel free to accept the suggestion to show the content in the PR for an easier way for WG to leave feedback.

**Comment by zijianjoy** [comment]
Kubeflow Pipelines roadmap for 2022: https://docs.google.com/presentation/d/18NLhNTHNKywSX3sOGjCcXGiVtqX-o9EuO7oWN9ECPtY/edit?resourcekey=0-Pwz21btM3HhG53ZBY14ToA#slide=id.gfbb06f32f0_0_692

**Comment by thesuperzapper** [comment]
@jbottum @annajung I think its mostly good, just needs some formatting fixes:

1. I think we should remove the `x.y.z` suffix, as we don't know what versions will be included.
     - we should also remove: `(note: Individual WG versions are independent of Kubeflow's)` as a result
1. I think we should add a "themes" header (like we did for 1.5), and add: `Support for Kubernetes v1.22` (rather than having it at the bottom of the page)

**Review by thesuperzapper — CHANGES_REQUESTED** [review]
@jbottum @annajung I think its mostly good, just needs some formatting fixes:

1. I think we should remove the `x.y.z` suffix, as we don't know what versions will be included.
     - we should also remove: `(note: Individual WG versions are independent of Kubeflow's)` as a result
1. I think we should add a "themes" header (like we did for 1.5), and add: `Support for Kubernetes v1.22` (rather than having it at the bottom of the page)

**Comment by jbottum** [comment]
@thesuperzapper @annajung I have updated per Mathew's suggestions.  @zijianjoy would you please review / approve.  Thanks!

File: ROADMAP.md [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+#### Themes
+* Kubernetes 1.22 support
+
+### Major Features from each Working Group (note: Individual WG versions are independent of Kubeflow's)
+
+#### Kubeflow Pipelines
+* V2 Preview: Support running pipeline in Argo-agnostic approach
+* New DAG visualization based on Pipeline Template and MLMD
+* SDK to change from `kfp.v2` to `kfp`
+* KFP SDK CLI design
```
Comment (zijianjoy) [inline ROADMAP.md:15]: Please add a link to this item: https://docs.google.com/document/d/1Z79ebKaf3hjyadSRsme1Qo7F-Tqu4qM1cyKuLeu2PgA/edit#heading=h.x9snb54sjlu9

Comment (jbottum) [reply ROADMAP.md]: added

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you so much Josh!

**Pull request merged**